fido-uaf-core/src/main/java/com/nexenio/fido/uaf/core/message/MatchCriteria.java

```diff
@@ -32,4 +33,32 @@
     @JsonProperty("aaid")
     private String[] aaids;
 
+    @SerializedName("vendorID")
+    @JsonProperty("vendorID")
+    private List<String> vendorIDs;
```

Comment:
Rename to `vendorIds` for consistency (like in `FinalChallengeParams.java`) and to meet [conventions](https://stackoverflow.com/questions/1699944/what-is-correct-java-naming-convention-for-id).